### DISCUSSION TITLE
Hi, i need help with Commitizen package!

### DISCUSSION THREAD

**Discussion by serdar-webdev** [community/community#125058] [npm] [answered]
Hi guys, I have one question, I use this https://commitizen-tools.github.io/commitizen/ package as git commit tool. I use it as dev-dependencies in my each project. And i don't want that for each project install package. Can i install it globally or something and use it without installing it into a project.

**Comment by SamiraPourkhaligh** [comment, marked as answer]
Yes, you can install the Commitizen package globally on your system so that you can use it across multiple projects without installing it in each project individually. Here's how you can do it:

1. Install Commitizen globally using npm:
```
bash
npm install -g commitizen
```
2. Once installed globally, you can use Commitizen in any Git repository by running the following command in the repository:
```
bash
git cz
```
This will launch Commitizen's interactive commit tool for creating standardized commit messages.

By installing Commitizen globally, you can avoid the need to include it as a dev-dependency in each project, making it easier to maintain and use across your projects.

**Reply by grandmas-favorite** [reply]
Nice one @SamiraPourkhaligh!
